benefits/core/views.py

```diff
@@ -31,20 +32,25 @@ def index(request):
     """View handler for the main entry page."""
     session.reset(request)
 
-    # generate a button to the landing page for each active agency
     agencies = models.TransitAgency.all_active()
-    buttons = [viewmodels.Button.outline_primary(text=a.short_name, url=a.index_url) for a in agencies]
-    buttons[0].classes.append("mt-3")
-    buttons[0].label = _("core.pages.index.chooseprovider")
 
-    page = viewmodels.Page(
-        title=_("core.pages.index.title"),
-        content_title=_index_content_title(),
-        buttons=buttons,
-        classes="home",
-    )
-
-    return PageTemplateResponse(request, page)
+    if len(agencies) == 1:
+        agency = agencies[0]
+        return redirect(agency.index_url)
+    else:
```

Comment:
I almost think we could live without this `else`. 

The previous `return` will end the flow, so don't really need to worry about branching the `if`.

Not a blocker.